pkg/lib/authn/otp/errors.go

```diff
@@ -12,7 +12,7 @@ var ErrCodeNotFound = InvalidOTPCode.NewWithCause("otp code is expired or invali
 var ErrInvalidCode = InvalidOTPCode.NewWithCause("invalid otp code", apierrors.StringCause("InvalidCode"))
 var ErrConsumedCode = InvalidOTPCode.NewWithCause("used otp code", apierrors.StringCause("UsedCode"))
 
-var ErrInvalidWhatsappUser = InvalidOTPCode.NewWithCause("invalid whatsapp user", apierrors.StringCause("InvalidWhatsappUser"))
+var ErrInvalidWhatsappUser = InvalidWhatsappUser.NewWithCause("invalid whatsapp user", apierrors.StringCause("InvalidWhatsappUser"))
```

Comment:
No need to specify cause here. Just call `New("invalid whatsapp user")`

Reply:
@louischan-oursky Updated, thanks!